account/build.gradle

```diff
@@ -17,6 +23,18 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-web'
 	runtimeOnly 'mysql:mysql-connector-java'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+	compile group: 'org.mockito', name: 'mockito-core', version: '3.6.28' // JUnit dependencies
+	testCompile('org.springframework.boot:spring-boot-starter-test')
+	testCompile('org.junit.jupiter:junit-jupiter:5.7.1')
+	testImplementation('junit:junit:4.13')
+	implementation 'org.springframework.boot:spring-boot-starter-actuator'
+	implementation 'org.springframework.boot:spring-boot-starter-data-rest'
+	compileOnly 'org.projectlombok:lombok'
+	developmentOnly 'org.springframework.boot:spring-boot-devtools'
```

Comment:
why's this needed?

Reply:
The spring-boot-devtools module is included to provide additional development-time features. To include devtools support, add the module dependency to the build, Since it is not mandatory removed it for now.